src/char/mapif.c

```diff
@@ -1816,29 +1816,30 @@ static int mapif_save_guild_storage_ack(int fd, int account_id, int guild_id, in
 
 /**
  * Loads the account storage and send to the map server.
- * @packet 0x3805     [out] <account_id>.L <struct item[]>.P
+ * @packet 0x3805     [out] <packet_len>.W <account_id>.L <storage_id>.W <struct item[]>.P
  * @param  fd         [in]  file/socket descriptor.
  * @param  account_id [in]  account id of the session.
```

Comment:
Add parameters `storage_id` and `storage_size`.

Reply:
done